keyboards/montsinger/rebound/rev5/info.json

```diff
@@ -0,0 +1,159 @@
+{
+    "manufacturer": "Montsinger",
+    "keyboard_name": "montsinger/rebound/rev5",
+    "maintainer": "rossman360",
+    "board": "GENERIC_RP_RP2040",
+    "bootloader": "rp2040",
+    "diode_direction": "ROW2COL",
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP0", "pin_b": "GP1"}
+        ]
+    },
+    "features": {
+        "audio": false,
+        "backlight": false,
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": true,
+        "extrakey": true,
+        "mousekey": false,
+        "nkro": false,
+        "oled": false,
+        "rgb_matrix": false,
+        "rgblight": false
+    },
+    "matrix_pins": {
+        "cols": ["GP6", "GP7", "GP8", "GP9", "GP23", "GP20", "GP22"],
+        "rows": ["GP2", "GP3", "GP4", "GP5", "GP26", "GP27", "GP28", "GP29"]
+    },
+    "mouse_key": {
+        "enabled": false
+    },
+    "processor": "RP2040",
+    "qmk": {
+        "tap_keycode_delay": 10
+    },
+    "rgb_matrix": {
+        "driver": "WS2812"
```

Comment:
If RGB Matrix is enabled this board will not compile; it's missing the LED count define in config.h and the LED config object here.